uplink/src/base/bridge/mod.rs

```diff
@@ -465,9 +489,9 @@ pub struct BridgeTx {
 impl BridgeTx {
     pub async fn register_action_route(&self, route: ActionRoute) -> Receiver<Action> {
         let (actions_tx, actions_rx) = bounded(1);
-        let ActionRoute { name, timeout } = route;
+        let ActionRoute { name, timeout, resend } = route;
```

Comment:
Is there any utility for generalizing resends?

Reply:
The configurations aren't exposed to the user, had decision paralysis with regards to clean-up. Have taken this as impetus to perform the same in latest commits.

Reply:
BTW, behavior remains unchanged after the following commits